themes/default/css/index.css

```diff
@@ -5383,6 +5383,7 @@ a.help .icon {
 /* This fixes a few outliers. */
 .icon-top {
 	vertical-align: middle;
+	padding-bottom: 3px;
```

Comment:
I am putting in a proper fix for this soon, before you get even more carried away : p